locale/fi/heading.json

```diff
@@ -0,0 +1,117 @@
+{
+  "title": "Otsikko (Heading)",
+  "intro": "Otsikot auttavat luomaan sivun sisällölle selkeän rakenteen, jonka avulla käyttäjä muodostaa kokonaiskäsityksen sivun sisällöstä.",
+  "note.title": "Saavutettavuus",
+  "note.items": [
+    "Kuten näkevä käyttäjä, ruudunlukijakäyttäjäkin muodostaa kokonaiskäsityksen sisällöstä otsikoiden perusteella.",
+    "Otsikointihierarkian tulee olla yhtenäinen (h1, h2, h3, h4, h5), eikä välistä saa puuttua otsikkotasoja. Oikea otsikkotasojen hierarkia tukee rakenteen hahmottamista.",
+    "Otsikkotasojen puuttuminen välistä tai otsikkotasojen epäjärjestys vaikeuttaa sisällön hahmottamista ruudunlukijaa käytettäessä. Ruudunlukija tulkitsee vain heading-merkityt komponentit otsikoiksi.",
+    "Huomioi tekstiä koskevia WCGA 2.1 ohjeistuksia: 1.4.3 Contrast (Minimum), 4.4 Resize text, 1.4.12 Text Spacing.",
```

Comment:
Should there be some link to WCGA-site instead of only mentioning the places in here?

And the place where to found this should be `1.4.4` for the Resize Text, it is currently missing the **1** from the front.

Links could be following:


```
1.4.3 Contrast (Minimum)
https://www.w3.org/TR/WCAG21/#contrast-minimum

1.4.4 Resize text
https://www.w3.org/TR/WCAG21/#resize-text

1.4.12 Text Spacing
https://www.w3.org/TR/WCAG21/#text-spacing
```

Or just one link like in the `styles/typography/`

```
WCAG 2.1 vaatimukset
https://www.w3.org/TR/WCAG21/
```